metrics/apis/metrics/handlers.go

```diff
@@ -26,14 +26,14 @@ import (
 	restful "github.com/emicklei/go-restful"
 	"github.com/golang/glog"
 
+	kube_v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	v1listers "k8s.io/client-go/listers/core/v1"
-	kube_v1 "k8s.io/client-go/pkg/api/v1"
-	"k8s.io/heapster/metrics/apis/metrics/v1alpha1"
 	"k8s.io/heapster/metrics/core"
 	metricsink "k8s.io/heapster/metrics/sinks/metric"
+	"k8s.io/metrics/pkg/apis/metrics/v1beta1"
```

Comment:
Nope, we can't do this -- the legacy client still expects the alpha API version on the non-aggregated endpoint.  If we're going to change this, we'd have to just remove it.  We can't really do that unless we also say that the most recent version of Heapster is the last to be supported as a source of metrics for HPA (FWIW, I'm fine with saying that, *but it's something that we need to decide as a team* @kubernetes/heapster-maintainers ).

Reply:
> the legacy client still expects the alpha API version on the non-aggregated endpoint.

The client is HPA?

/cc @piosz @loburm

Reply:
/cc @MaciekPytel @kawych to address the HPA question.

Reply:
The legacy client is used by HPA and kubectl top. The latter case was removed ~1 week ago in #56206. I agree with @DirectXMan12 that we shouldn't change the API version, instead we should remove this endpoint entirely from Heapster at some point (I'd say it's definetely too early for that).

Reply:
+1 for not bumping the version, as kubectl top relied on this until very recently.